samples/audio-recording-add-on/src/index.js

```diff
@@ -0,0 +1,188 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import AddOnSdk from "https://new.express.adobe.com/static/add-on-sdk/sdk.js";
```

Comment:
```suggestion
import addOnUISdk from "https://new.express.adobe.com/static/add-on-sdk/sdk.js";
```
Lets use `addOnUISdk` as suggested in [documentation](https://developer.adobe.com/express/add-ons/docs/references/addonsdk/#import-into-javascripttypescript-file)

Please also make this change in other sample add-ons.